content/docs/trust/trust-manager/README.md

```diff
@@ -88,18 +91,25 @@ spec:
 - `inLine` - a manually specified string containing at least one certificate
 - `useDefaultCAs` - usually, a bundle of publicly trusted certificates
 
-These sources, along with the single currently supported target type (`configMap`)
-are documented in the trust-manager [API reference documentation](./api-reference.md).
+`ConfigMap` is the default target type, but as of v0.7.0 trust-manager also supports `Secret` resources as targets.
+
+Support for `Secret` targets must be explicitly enabled in the trust-manager controller; see details below under "Enable Secret targets".
+
+All sources and target options are documented in the trust-manager [API reference documentation](./api-reference.md).
```

Comment:
@SgtCoDFish Do we have CI to update API docs in this repo? It seems like it currently misses the v0.7.0 extensions of the Bundle API.

Reply:
It requires a [manual bump here](https://github.com/cert-manager/website/blob/master/scripts/gendocs/generate-trust-manager#L62).

It probably _shouldn't_ require a manual bump, but it does #:grin: